src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -15,9 +15,9 @@
       <!-- This specifies what properties are needed to be passed down as global properties to a child project. -->
 
       <_PropertiesToPass>
-        TargetArchitecture=$(TargetArchitecture);
-        TargetOS=$(TargetOS);
-        Configuration=$(Configuration);
+        TargetArchitecture=$(__BuildArch);
+        TargetOS=$(__TargetOS);
```

Comment:
We agreed yesterday that these properties should be passed globally as we set them in various places to default values, so to avoid discrepancies. So we should continue using `TargetArchitecture`, `TargetOS` and `Configuration`

Reply:
Hmm, I'm not sure if I'm getting what you're suggesting. Are you saying that we should rename the three environment variables in send-to-helix-step, perhaps to something like <code>_TargetArchitecture</code>, <code>_TargetOS</code> and <code>_Configuration</code> in accordance with the other variables? Or that we shouldn't be setting these variables in the helixpublishwitharcade.proj script? If it's the latter, how are we going to propagate them to the script from the YML pipeline?

Reply:
I'm saying we don't use a different name, use the normal name and just pass them globally when running msbuild helixpublishwitharcade.proj /p:TargetArchitecture=... directly in the scripts or the yml.

I looked at a binlog yesterday and TargetOS is set based on the machine host, so it is better to set it globally from the command line rather than fighting against our infra to set all these values to a default value if it is not set globally.

Reply:
Thanks for clarifying, hopefully fixed in 6th commit.